portfolio/src/main/webapp/map.js

```diff
@@ -17,11 +17,144 @@
  */
 window.addEventListener('load', createMap);
 
+
+const icons = {
+  image: {
+  name: 'Image Locations',
+  icon: 'https://raw.githubusercontent.com/Concept211/Google-Maps-Markers/master/images/marker_red.png'
+  },
+  place: {
+  name: 'Places To Go',
+  icon: 'https://raw.githubusercontent.com/Concept211/Google-Maps-Markers/master/images/marker_green.png'
+  },
+  user: {
+  name: 'User Submitted',
+  icon: 'https://raw.githubusercontent.com/Concept211/Google-Maps-Markers/master/images/marker_purple.png'
+  }
+};
+
 /** 
  * Creates a map and adds it to the page. 
  */
 function createMap() {
   const map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+      {center: {lat: 36.9, lng: -76.66}, zoom: 5.2}); 
+
+  // add hardcoded markers to visualize where each gallery image was taken
+  addImageLandmark(
```

Comment:
A few of these are missing `;`

https://google.github.io/styleguide/jsguide.html#formatting-function-arguments

The style you've chosen is CORRECT and thus this note is not a required change. These examples are also from function declaration notes not function calling, but....

```javascript
// ... Uses less vertical space,
// but violates the rectangle rule and is thus not recommended.
```

```javascript
// Four-space, one argument per line.  Works with long function names,
// survives renaming, and emphasizes each argument.
doSomething(
    veryDescriptiveArgumentNumberOne,
    veryDescriptiveArgumentTwo,
    tableModelEventHandlerProxy,
    artichokeDescriptorAdapterIterator) {
  // …
}
```